entrypoint.sh

```diff
@@ -6,5 +6,6 @@ set -e
 for tb in /test-binaries/*
 do
     echo "Running tests in: $tb"
-    $tb
+    # Always exit successfully so that the test job isn't restarted
+    $tb || exit 0
```

Comment:
with this change how do we know if the tests fail?

Reply:
Yeah, this was a tough choice. If we let the tests exit with a failure, that causes the container to restart because we're using `restartPolicy: OnFailure`. If we don't use this `restartPolicy`, there are other issues.